packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -31,7 +32,12 @@ import { IRequest } from "@fluidframework/core-interfaces";
 
 const mapId = "map";
 const stringId = "sharedStringKey";
-const registry: ChannelFactoryRegistry = [[mapId, SharedMap.getFactory()], [stringId, SharedString.getFactory()]];
+const cellId = "cellKey";
+const registry: ChannelFactoryRegistry = [
+    [mapId, SharedMap.getFactory()],
+    [stringId, SharedString.getFactory()],
+    [cellId, SharedCell.getFactory()]];
```

Comment:
This looks right. It might be better (and easier) to add unit tests to `packages/dds/cell/src/test/cell.spec.ts` instead though.

But I think it's also fine to add tests here. They should look similar to the "resends op"/"doesn't resend successful op" tests below, except using SharedCell instead of SharedMap.